xscen/diagnostics.py

```diff
@@ -395,9 +402,12 @@ def properties_and_measures(
                 sim=prop[vname], ref=dref_for_measure[vname]
             )
             # create a merged long_name
-            prop_ln = prop[vname].attrs.get("long_name", "").replace(".", "")
```

Comment:
we are loosing the `.replace(".", "")`

Reply:
I assumed that this `replace` was to remove a potential trailing dot ? I changed `update_attr` to strip all leading and trailing whitespace and dots instead.